packages/beacon-node/src/chain/blocks/types.ts

```diff
@@ -21,63 +21,94 @@ export enum BlockSource {
   byRoot = "req_resp_by_root",
 }
 
+export enum GossipedInputType {
+  block = "block",
+  blob = "blob",
+  dataColumn = "dataColumn",
+}
+
+interface CachedDataItem {
+  cacheId: number;
+}
+interface Availability<T> {
+  availabilityPromise: Promise<T>;
+  resolveAvailability: (data: T) => void;
+}
+
+/**
+ *
+ * Deneb Blob Format Types
+ *
+ */
 /** Enum to represent where blobs come from */
 export enum BlobsSource {
   gossip = "gossip",
   api = "api",
   byRange = "req_resp_by_range",
   byRoot = "req_resp_by_root",
 }
+interface ForkBlobsInfo {
```

Comment:
prefer "type" over "interface" to model types of objects

Reply:
updated